content/courses/connecting-to-offchain-data/verifiable-randomness-functions.md

```diff
@@ -514,7 +519,7 @@ pub mod instructions;
 pub mod state;
 pub mod errors;
 
-declare_id!("YOUR_KEY_HERE");
+declare_id!("YOUR_KEY_HERE"); // Replace with your program key
```

Comment:
Honestly people can read code, they don't need the comment, maybe just:

```suggestion
declare_id!("YOUR_PROGRAM_PUBLIC_KEY_HERE"); 
```

To make sure people know it's the public key (which should be obvious but this is training content and we should be explicit).

You could also tell people to run `anchor keys sync` if necessary.